### ISSUE TITLE
Doesn't Work: Blank Screen

### ISSUE THREAD

**Issue by Marioh2451** [lencx/nofwl#74]
Somehow reopening the program NoFWL because i've got issues with the API key and i was getting error 429, the program returns itself with a blank screen.
I've tried refreshing it, uninstalling and installling again, rebooting my computer, i don't know what to do.
![imagen](https://github.com/lencx/nofwl/assets/40912167/689b1ab7-995a-4738-8e85-0465afa7d0de)

**Comment by ahahaber** [comment]
I have the same issue

**Comment by xxl-cc** [comment]
I have the same issue

**Comment by ares0027** [comment]
yesterday it was working, today, this issue

**Comment by Piscabo** [comment]
I thought I just installed and I broke it already... Glad to see it's not just me. Any fix at hand? ![image](https://github.com/lencx/nofwl/assets/123719842/bc79d34d-cde8-42d5-810d-7eaa8427a948)

**Comment by lysergicstationer** [comment]
Is there any fix for this yet?

**Comment by opcode81** [comment]
Same issue; I switched to using [slickgpt](https://github.com/ShipBit/slickgpt)

**Comment by sesey** [comment]
yes i am having the same problem. The problem is due to the inability to load html, css or script content, and to temporarily overcome this problem, when you right click and click on the inspect section, the developer tool of the edge browser will open and when we refresh the page, the problem is solved.
